### DISCUSSION TITLE
Convert latitude/longitude coordinates

### DISCUSSION THREAD

**Discussion by yooakim** [duckdb/duckdb#11030] [Q&A]
I have a file with latitude and longitude in a single column as this:
```
60° 20.908, 18° 27.584
```
These values are in Degrees Minutes Seconds (D° M' S") format.

To make using them easier I want to convert them to Decimal Degrees (D.D°)

```
45.7811111 -108.5038888
```

I can do the conversion but I am wondering what would be the _best_ way to do this with DuckDb? 

Would it make more sense to fix this before importing to DuckDb or could it be done as part of the ingestion?

Any ideas are very welcome!

And I think I am not alone in seeing a need to convert between the three most used formats of latitude/longitude -  Degrees Minutes Seconds (D° M' S"), Decimal Minutes (D° M.M'), and Decimal Degrees (D.D°)

**Comment by NickCrews** [comment]
I would just do this duckdb, it isn't that hard to express and should be plenty fast.

**Comment by yooakim** [comment]
After some dialogue on Discord I've added a new [issue](https://github.com/duckdb/duckdb_spatial/issues/278) in the spatial extensions repo. Closing this.
